sdk/tests/client/signing/account.rs

```diff
@@ -202,6 +206,7 @@ async fn account_reference_unlocks() -> Result<()> {
         )
         .with_outputs(outputs)
         .with_creation_slot(slot_index + 1)
+        .with_capabilities([TransactionCapabilityFlag::BurnMana])
```

Comment:
Why are we adding that?

Reply:
I tried removing them and tests are failing

Reply:
But since it's signing tests, maybe it's not that bad to keep them since it's not really the point of the test

Reply:
Yeah one would need to adjust them if it should pass without it